xbmc/pictures/Picture.cpp

```diff
@@ -214,12 +214,13 @@ bool CPicture::CacheTexture(uint8_t *pixels, uint32_t width, uint32_t height, ui
   uint32_t max_height = advancedSettings->m_imageRes;
   if (advancedSettings->m_fanartRes > advancedSettings->m_imageRes)
   { // 16x9 images larger than the fanart res use that rather than the image res
-    if (fabsf(static_cast<float>(width) / static_cast<float>(height) / (16.0f / 9.0f) - 1.0f) <= 0.01f &&
-        height >= advancedSettings->m_fanartRes)
+    if (fabsf(static_cast<float>(width) / static_cast<float>(height) / (16.0f / 9.0f) - 1.0f)
+        <= 0.01f)
```

Comment:
well done! imo the following suggestion by clang-format
```suggestion
    if (fabsf(static_cast<float>(width) / static_cast<float>(height) / (16.0f / 9.0f) - 1.0f) <=
        0.01f)
```
can be ignored. @phunkyfish can tell us :)

Reply:
I think either is ok. But as you are changing it anyway by squashing it’s up to you 😉

Reply:
it‘s already squashed 😀 @roidy we‘re good i think